flink-table-store-connector/src/main/java/org/apache/flink/table/store/connector/sink/FlinkSink.java

```diff
@@ -103,7 +103,7 @@ public DataStreamSink<?> sinkFrom(
         CommittableTypeInfo typeInfo = new CommittableTypeInfo();
         SingleOutputStreamOperator<Committable> written =
                 input.transform(
-                                WRITER_NAME,
+                                WRITER_NAME + " -> " + table.name(),
```

Comment:
We can add name to committer too.

Reply:
Has been completed in [FLINK-31094] Add table name to Committer